src/libcore/mem.rs

```diff
@@ -177,15 +177,57 @@ pub fn forget<T>(t: T) {
 
 /// Returns the size of a type in bytes.
 ///
-/// More specifically, this is the offset in bytes between successive
-/// items of the same type, including alignment padding.
+/// More specifically, this is the offset in bytes between successive elements
+/// in an array with that item type including alignment padding. Thus, for any
+/// type `T` and length `n`, [T; n] will have a size of `n * size_of<T>()`. 
```

Comment:
`[00:03:45] tidy error: /checkout/src/libcore/mem.rs:182: trailing whitespace`

Reply:
and `size_of::<T>()` here as well